addons/new-osio-addon/src/main/java/io/fabric8/launcher/osio/projectiles/OsioProjectile.java

```diff
@@ -8,6 +8,9 @@
  */
 @Value.Immutable
 public interface OsioProjectile extends CreateProjectile {
+
+    String getGitOrganization();
```

Comment:
That's ok, perhaps we should consider moving that to `CreateProjectile`

Reply:
atm we don't support changing the organization for launch, but we could